config/esg_config.yaml

```diff
@@ -0,0 +1,127 @@
+
```

Comment:
Does this  `esg_config.yaml` file need to be in the commit? Isn't it generated at each run?

Reply:
It doesn't need to be part of the commit.  I didn't realize it was included

Reply:
Oh I see what happened. I had another copy of esg_config.yml in my config/ directory. Was playing around with that a while ago and never cleaned it up.